src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -173,4 +186,15 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
       this.queriesResults
     );
   }
+
+  private getTelemetryReport() {
+    const observables = [
+      this.mgrModuleService.getOptions('telemetry'),
+      this.mgrModuleService.getConfig('telemetry')
+    ];
+    observableForkJoin(observables).subscribe((resp: object) => {
+      const configResp = resp[1];
+      this.telemetryEnabled = configResp?.enabled;
+    });
+  }
```

Comment:
do we need to subscribe to these two observables? I think just the `getConfig` would be enough. It has the `enabled` option.

```suggestion
    this.mgrModuleService.getConfig('telemetry').subscribe((resp: object) => {
         this.telemetryEnabled = resp?.enabled
      });
```

Reply:
@cloudbehl do we need both?

Reply:
I added the changes, forgot to push it. Now done